src/box/lua/load_cfg.lua

```diff
@@ -83,6 +83,8 @@ local default_cfg = {
     flightrec_requests_max_req_size = 16384,
     flightrec_requests_max_res_size = 16384,
 
+    wal_ext = nil,
+
```

Comment:
Let's put this option together with other `wal_` options.

Reply:
Moved.